dev-docs/bidders/onetag.md

```diff
@@ -5,7 +5,7 @@ description: Prebid OneTag Bidder Adaptor
 pbjs: true
 pbs: true
 biddercode: onetag
-media_types: banner, video
+media_types: banner, video, native
```

Comment:
Can you add the new meta data field  `multiformat_supported`. It takes one of the three following values

* will-bid-on-any
* will-bid-on-one
* will-not-bid

See https://docs.prebid.org/formats/multiformat.html for more info